pyproject.toml

```diff
@@ -73,6 +73,7 @@ complete = [
 ]
 test = [
     "pandas[test]",
+    "awkward",
```

Comment:
-1 from me. I'd rather not add test dependencies to our already heavy test suite.
Also this doesn't actually run the added test in CI.